hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -229,21 +230,26 @@ N.B. NETCONSOLE blocks until the ethernet link is established or a timeout occur
 See https://wiki.archlinux.org/index.php/Netconsole
 ```
 src_port@src_ip/dev_name,dst_port@tgt_ip/tgt_mac
-E.g. 6665@169.254.1.1/eth0,6666@/
+E.g. 6665@169.254.1.1/,6666@/
 ```
-In order to simplify parsing, the bootloader requires every field separator to be present. In the example the target IP address (255.255.255.255) and target mac address (00:00:00:00:00) are assigned default values.
+In order to simplify parsing, the bootloader requires every field separator to be present. The source ip address must be specified but the following fields may be left blank and assigned default values.
+
+* src port - 6665
+* dst port - 6666
+* dst ip - 255.255.255.255
+* dst mac - 00:00:00:00:00
```

Comment:
The `src_port@src_ip/dev_name,dst_port@tgt_ip/tgt_mac` used above has a slight mismatch with the `dst ip` and `dst mac` names used here?